csharp/test/Apache.Arrow.Tests/Decimal256ArrayTests.cs

```diff
@@ -97,6 +97,22 @@ public void AppendLargeDecimal()
                     Assert.Equal(-large, array.GetValue(1));
                 }
 
+                [Fact]
+                public void AppendMaxAndMinDecimal()
+                {
+                    // Assert
+                    var builder = new Decimal128Array.Builder(new Decimal128Type(29, 0));
```

Comment:
This is the Decimal256 tests, but we are using Decimal128 here

```suggestion
                    var builder = new Decimal256Array.Builder(new Decimal128Type(29, 0));
```

Reply:
fixed